src/index.njk

```diff
@@ -12,8 +12,8 @@ hubspot:
     <div class="max-w-md sm:max-w-screen-lg mt-12 mx-auto">
         <div class="container m-auto text-left max-w-screen-lg lg:flex lg:flex-row lg:items-start lg:justify-between lg:gap-6">
             <div class="max-w-xl mx-auto lg:mx-0 lg:my-auto lg:py-10 lg:max-w-[600px]">
-                <h1 class="font-medium leading-tight max-w-xl m-auto lg:m-0 text-center lg:text-left text-indigo-800 text-4xl">
-                    {{ site.messaging.tagLine }}
+                <h1 class="font-medium leading-tight max-w-xl m-auto lg:m-0 text-center lg:text-left text-indigo-800 text-4xl sm:text-5xl lg:text-4xl">
```

Comment:
From `4xl` to `5xl`, I didn’t expect the change to be so significant. It looks enormous—I don’t think this is necessary...

Reply:
It is large, yes, I wanted it somewhere between, but this was still better than `4xl` imo at that screen size, makes it read as an actual hero-line, rather than just a smaller header. Happy to be overruled though

Reply:
Great! I did try it locally, and it didn’t seem to work for me. I’m glad it was as simple as that.

Reply:
Let's leave this for a separate PR if we decide to revisit the `h1` in future iterations.